packages/schemas/src/db-entries/custom-types.ts

```diff
@@ -5,6 +5,11 @@ export enum ApplicationType {
   SPA = 'SPA',
   Traditional = 'Traditional',
 }
+export enum LogType {
+  SignInSuccess = 'SignInSuccess',
+  SignInFailedWrongPassword = 'SignInFailedWrongPassword',
+  ExchangeAccessTokenSuccess = 'ExchangeAccessTokenSuccess',
```

Comment:
will it be better to separate into 2 or more types, for better analytics experience.

e.g.:

```ts
enum LogType {
  SignIn = 'SignIn',
  ExchangeToken = 'ExchangeToken',
}

enum LogResult {
  Success = 'Success',
  Failed = 'Failed',
}

enum TokenType {
  AccessToken = 'AccessToken',
  IDToken = 'IDToken',
}

enum SignInType {
  UsernameAndPassword = 'UsernameAndPassword',
  // ...
}

type LogType = TokenType | SignInType;
```

thus we can store `result` and `type` column in DB (nullable)

Reply:
Separated into `log_result` and `log_type`, but not split `LogType` into subtypes, because this is an autogenerated file actually.

Reply:
LOG-389